universal_transfer_operator/src/universal_transfer_operator/data_providers/filesystem/sftp.py

```diff
@@ -125,6 +134,17 @@ def write_using_smart_open(self, source_ref: FileStream) -> str:
             stream.write(source_ref.remote_obj_buffer.read())
         return complete_url
 
+    def write_dataframe_using_smart_open(self, source_ref: pd.DataFrame) -> str:
```

Comment:
How important is it to have in the function name how it is implemented? Do we have a `write_dataframe` that does not use smart open?

Reply:
Not important, maybe we can change it to - `write_from_dataframe()` and `write_from_stream()`